datafusion/physical-expr/src/analysis.rs

```
@@ -231,7 +231,8 @@ fn calculate_selectivity(
                 1.0,
                 |acc, (i, ExprBoundaries { interval, .. })| {
                     let temp =
-                        cardinality_ratio(&initial_boundaries[i].interval, interval)?;
+                        cardinality_ratio(&initial_boundaries[i].interval, interval)
```

This error is being silenced. Is it because of float division?

---

cardinality can be between 0.0 and 1.0. If we get an error, we just use 1.0 for cardinality estimate (safest estimate).